src/scripts/webpack/config.js

```diff
@@ -34,16 +39,22 @@ const readOptionalConfigFile = fullPath => {
 
 // try to load configuration type from the user project, fallback on BaseConfig
 const getConfigType = (configFolderPath) => {
-  const configTypePath = path.resolve(configFolderPath, './Config');
+  const configTypePath = path.resolve(configFolderPath, './Config.js');
   try {
+    if (!fs.existsSync(configTypePath)) {
+      /* eslint-disable no-console */
+      console.warn(`
+        No Config type definition file found in ${configTypePath}, using the default one...
+      `);
+      /* eslint-enable no-console */
+      return defaultConfigType;
+    }
     return require(configTypePath);
   } catch (e) {
     /* eslint-disable no-console */
-    console.warn(`
-      No Config type definition file found in ${configTypePath}, using the default one...
-    `);
+    console.error(`Invalid Config type definition file: ${configTypePath}`);
     /* eslint-enable no-console */
-    return t.Any;
+    process.exit(1);
```

Comment:
Minor, and I thought about this after our offline discussion, but: "Invalid Config" is a bit misleading, we are not validating the config with respect to a type here (yet). Also in this way we are not reporting the precise error to the user.

Instead, I'd propose to simplify this even further and:
- remove the `try/catch`
- keep the `if (!fs.existsSync)` block as it is now
- just plain `return require(configTypePath)`

In this way, if the file is not valid JS we just fail and report the original error to the user